clang/docs/ReleaseNotes.rst

```diff
@@ -264,6 +264,7 @@ Bug Fixes to C++ Support
 - Properly reject defaulted copy/move assignment operators that have a non-reference explicit object parameter.
 - Clang now properly handles the order of attributes in `extern` blocks. (#GH101990).
 - Fixed an assertion failure by preventing null explicit object arguments from being deduced. (#GH102025).
+- Fixed a crash caused by marking the function declaration as invalid during strictness checks on ``extern`` main. (#GH104570).
```

Comment:
Is it actually necessary, given that the issue was introduced in the same release?

Reply:
Good question. Should this be treated as a follow-up to the main PR, or listed separately as a fix? I've removed it for now, but I can revert it if needed...

Reply:
Release notes should list changes from the previous release. Since both the issue and the fix are within the same release (LLVM 20), there's no need to describe the fix separately. That's my understanding at least ;)